db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161222154024) do
+ActiveRecord::Schema.define(version: 20170104135212) do
```

Comment:
I doubt this is necessary since you didn't change anything. It just shows you the difference, because you ran the migrations at a later time. Or maybe you forgot to commit something?